weekly_report.yaml

```diff
@@ -10,11 +10,58 @@ teams:
+      Team Activity:
+        filename: rhdh-team-activity.pdf
+        url: https://logilica.io/standup?chartId=40f04dce-01fc-4d86-aebe-3d2be1a677ca&variables=W3sidmFyIjoiVGVhbS5uYW1lIiwidmFsIjp7Im1lbWJlciI6IlRlYW0ubmFtZSIsIm9wZXJhdG9yIjoiZXF1YWxzIiwidmFsdWVzIjpbIlJIREggQ29yZSBUZWFtIiwiUkhESCBEeW5hbWljIFBsdWdpbnMgVGVhbSIsIlJIREggSW5zdGFsbCBUZWFtIiwiUkhESCBQbHVnaW5zIFRlYW0iLCJSSERIIFBsdWdpbnMvQUkgLSBVSSBUZWFtIiwiUkhESCBTZWN1cml0eSBUZWFtIiwiUkhESCBVSSBUZWFtIl19fSx7InZhciI6IlRpbWUiLCJ2YWwiOnsiZGltZW5zaW9uIjoiVGltZSIsImRhdGVSYW5nZSI6ImZyb20gMzAgZGF5cyBhZ28gdG8gbm93In19XQ%3D%3D&workstream=00000000004efddf
+      # RHDH Team Dashboard:
+      #   filename: rhdh-teams-dashboard.pdf
+      #   url: https://logilica.io/dashboard/69f12907-3d84-4e8e-9a2a-1d4b4f5aeafc?variables=W3sidmFyIjoiVGltZSIsInZhbCI6eyJkaW1lbnNpb24iOiJUaW1lIiwiZGF0ZVJhbmdlIjoiZnJvbSA5MCBkYXlzIGFnbyB0byBub3cifX0seyJ2YXIiOiJKaXJhUHJvamVjdC5uYW1lIiwidmFsIjp7Im1lbWJlciI6IkppcmFQcm9qZWN0Lm5hbWUiLCJvcGVyYXRvciI6ImVxdWFscyIsInZhbHVlcyI6WyJpc3N1ZXMucmVkaGF0LmNvbS9SSERIQlVHUyIsImlzc3Vlcy5yZWRoYXQuY29tL1JIREhQQUkiLCJpc3N1ZXMucmVkaGF0LmNvbS9SSElEUCJdfX0seyJ2YXIiOiJUZWFtLm5hbWUiLCJ2YWwiOnsibWVtYmVyIjoiVGVhbS5uYW1lIiwib3BlcmF0b3IiOiJlcXVhbHMiLCJ2YWx1ZXMiOlsiUkhESCBDb3JlIFRlYW0iLCJSSERIIER5bmFtaWMgUGx1Z2lucyBUZWFtIiwiUkhESCBJbnN0YWxsIFRlYW0iLCJSSERIIFBsdWdpbnMgVGVhbSIsIlJIREggUGx1Z2lucy9BSSAtIFVJIFRlYW0iLCJSSERIIFNlY3VyaXR5IFRlYW0iLCJSSERIIFVJIFRlYW0iXX19XQ%3D%3D
```

Comment:
Are you sure you want to commit commented-out lines?

Reply:
If so, feel free to mark this as resolved and go ahead and merge.

Reply:
Well, this is now broken and pending fix on Logilica end. Uncommenting is much easier than finding the link again :-)